backend/controllers/createUser.js

```diff
@@ -45,6 +53,8 @@ const createUserFactory = (getAssignableGroupsApi, createUser, createUrl, manage
     } else {
       try {
         await createUser(res.locals, user)
+
+        req.session.searchResultsUrl = `${searchUrl}/results?user=${user.username}`
```

Comment:
special case - when creating a user the search results breadcrumb takes them to a search with just that user in the results